src/FixedDataTableRow.js

```diff
@@ -157,11 +170,18 @@ class FixedDataTableRowImpl extends React.Component {
         onColumnReorderEnd={this.props.onColumnReorderEnd}
         isColumnReordering={this.props.isColumnReordering}
         columnReorderingData={this.props.columnReorderingData}
-        rowHeight={this.props.height}
+        rowHeight={cellHeight}
         rowIndex={this.props.index}
       />;
     var scrollableColumnsWidth = this._getColumnsWidth(this.props.scrollableColumns);
     var columnsRightShadow = this._renderColumnsRightShadow(fixedColumnsWidth + scrollableColumnsWidth);
+    var rowExpandedStyle = {
+      top: cellHeight,
+      width: this.props.width - /* borderWidth */ 2,
+      height: this.props.height - this.props.cellHeight,
+      zIndex: 2
```

Comment:
I just followed fixedColumns has zIndex=2, now we have sub row, sure, we don't need it now